static/app/utils/discover/eventView.tsx

```diff
@@ -1259,6 +1257,23 @@ class EventView {
   }
 }
 
+const isFieldsSimilar = (
+  currentValue: Array<string>,
+  otherValue: Array<string>
+): boolean => {
+  const currentEquations = currentValue.filter(isEquation);
+  const otherEquations = otherValue.filter(isEquation);
```

Comment:
This is a list because order matters right? Can we leave a comment or something so it's explicit